<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>158</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate velocities</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Planet</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mercury</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Venus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Earth</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mars</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jupiter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturn</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Uranus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Neptune</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Own</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass [Yg]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius [km]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance [AU]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Center object</string>
   </property>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sun</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Own</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass [Yg]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius [km]</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sagittarius A*</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxima Centauri System</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>481</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Results</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>First cosmic velocity (orbital velocity)</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km above the object surface:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km above the object surface:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_500">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>500</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_200">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5000">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5 000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km above the object surface:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km above the object surface:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10000">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10 000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km above the object surface:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::RightToLeft</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_201">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Altitude</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Second cosmic velocity (escape velocity)</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min. escape velocity from surface of the object:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Third cosmic velocity (solar system escape velocity)</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min. system escape velocity from object surface:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 km/s</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>158</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export data</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
